docs/en/observability/apm/open-telemetry.asciidoc

```diff
@@ -1,51 +1,129 @@
+such as which sources are collected by default.
 
-To unlock the full power of the {stack}, use the OpenTelemetry API/SDKs with Elastic APM agents,
-currently supported by the Java, Python, .NET, and Node.js agents.
-These Elastic APM agents translate OpenTelemetry API calls to Elastic APM API calls.
-This allows you to reuse your existing instrumentation to create Elastic APM transactions and spans--
-avoiding vendor lock-in and having to redo manual instrumentation.
+// TODO: Update links when distro docs are markdown-ified
+Get started with an Elastic Distribution for OpenTelemetry:
 
-<<apm-open-telemetry-with-elastic,Get started>>.
+* https://github.com/elastic/elastic-otel-java[Java]
+* https://github.com/elastic/elastic-otel-dotnet[.NET]
+* https://github.com/elastic/elastic-otel-node[Node.js]
+* https://github.com/elastic/elastic-otel-python[Python]
 
-**OpenTelemetry agent**
+[NOTE]
+====
+For more details about OpenTelemetry distributions in general,
+visit the https://opentelemetry.io/docs/concepts/distributions[OpenTelemetry documentation].
+====
+
+[float]
+[[apm-otel-api-sdk]]
+== OpenTelemetry API/SDK
 
 The {stack} natively supports the OpenTelemetry protocol (OTLP).
 This means trace data and metrics collected from your applications and infrastructure by an
 OpenTelemetry agent can be sent directly to the {stack}.
 
-<<apm-instrument-apps-otel,Get started>>.
+// Do we want to say anything about why you would/wouldn't choose this
+// method to send data to Elastic?
+
+<<apm-instrument-apps-otel,*Get started with an OpenTelemetry API/SDK →*>>
+
+image::images/apm-otel-api-sdk.png[]
 
-**OpenTelemetry collector**
+{nbsp} +
 
+[float]
+[[apm-otel-collector]]
+== OpenTelemetry Collector
+
+// Should we also mention the Elastic Distribution for OpenTelemetry Collector?
 The {stack} natively supports the OpenTelemetry protocol (OTLP).
 This means trace data and metrics collected from your applications and infrastructure by an
-OpenTelemetry collector can be sent directly to the {stack}.
+OpenTelemetry Collector can be sent directly to the {stack}.
+
+// Do we want to say anything about why you would/wouldn't choose this
+// method to send data to Elastic?
 
-<<apm-connect-open-telemetry-collector,Get started>>.
+<<apm-connect-open-telemetry-collector,*Get started with an OpenTelemetry Collector →*>>
 
-**Lambda collector exporter**
+image::images/apm-otel-collector.png[]
+
+{nbsp} +
+
+[float]
+[[apm-otel-api-sdk-elastic-agent]]
+== OpenTelemetry API/SDK + Elastic APM agent
+
+Use the OpenTelemetry API/SDKs with Elastic APM agents to translate OpenTelemetry API calls to Elastic APM API calls.
+This allows you to reuse your existing instrumentation to create Elastic APM transactions and spans--avoiding
+vendor lock-in and having to redo manual instrumentation.
+
+// Do we want to say anything about why you wouldn't choose this
+// method to send data to Elastic?
```

Comment:
We kind of get into why you _would_ choose this method in the first paragraph. Do we want to provide any guidance on when you would _not_ want to use this method, too?